examples/tests/sdk_tests/std/array.test.w

```diff
@@ -73,6 +73,8 @@ let item = a.pop();
 assert(item == "world");
 assert(a.length == 1);
 assert(a.at(0) == "hello");
+let pushMultipleItems: MutArray<str> = MutArray<str> ["element1"];
+pushMultipleItems.push("element2", "element3");
 
 test "pushAndPop()" {
   let a = MutArray<str>["hello"];
```

Comment:
After you add the assert statements, please copy all the code for the test you added to this test scope. It may seem like a repetition, but this is how we currently test that the .push method also works inflight.

Reply:
Added the assert statements to both the preflight and inflight contexts